website/docs/cdktf/test/unit-tests.mdx

```diff
@@ -5,28 +5,44 @@ description: Write assertions and snapshot tests for your CDK for Terraform appl
 
 # Unit Tests
 
-Testing your application can give you faster feedback cycles and guard you against unwanted changes. Currently, testing is only supported in Typescript with jest.
+Testing your application can give you faster feedback cycles and guard you against unwanted changes. Testing is currently supported in Typescript with jest and compatible with any testing framework that supports assertions for all other languages.
 
-We generate all files necessary to run jest when you run `cdktf init` so that you can start writing tests right away. If you want to add jest to an existing project, please [follow their guide](https://jestjs.io/docs/getting-started). Then, you need to add these lines in a [setup file](https://jestjs.io/docs/configuration#setupfiles-array):
+We generate all files necessary to start testing when you run `cdktf init` so that you can start writing tests right away.
+
+If you want to add jest to an existing Typescript project, please [follow their guide](https://jestjs.io/docs/getting-started). Then, you need to add these lines in a [setup file](https://jestjs.io/docs/configuration#setupfiles-array):
 
 ```js
 const cdktf = require("cdktf");
 cdktf.Testing.setupJest();
 ```
 
+If you want to add pytest to an existing Python project, please [follow their guide](https://docs.pytest.org/en/7.1.x/getting-started.html). The CDKTF specific assertions can be found in the cdktf package under `Testing`.
+
+If you want to add JUnit to an existing Java project, please [follow their guide](https://maven.apache.org/plugins-archives/maven-surefire-plugin-2.12.4/examples/junit.html). The CDKTF specific assertions can be found in the cdktf package under `Testing`.
+
+If you want to add Xunit to an existing C# project, please [follow their guide](https://xunit.net/docs/getting-started/netcore/cmdline). The CDKTF specific assertions can be found in the cdktf package under `Testing`.
+
+If you want to add testing to an existing Go project, please [follow their guide](https://go.dev/doc/tutorial/add-a-test). The CDKTF specific assertions can be found in the cdktf package under `Testing`.
```

Comment:
Okay how do we feel about making this an actual section? I think it's generally about how to set up testing and integrate it into your project, right? (I realize there wasn't a section before, so this is new!)

Also - when we say that we autogenerate all the files, is that true for other testing frameworks, or is it just true for Jest with TypeScript?

```suggestion
## Add Testing to Your Application

Use the following resources to add testing to your CDKTF application:
- **TypeScript:** Follow TypeScript's [Getting Started guide](https://jestjs.io/docs/getting-started) for testing with Jest. Then, add these lines in a [setup file](https://jestjs.io/docs/configuration#setupfiles-array):

    ```js
    const cdktf = require("cdktf");
    cdktf.Testing.setupJest();
    ```
- **Python:** Follow the [Get Started guide](https://docs.pytest.org/en/7.1.x/getting-started.html) for pytest. The assertions for CDKTF are available in the `cdktf` package under `Testing`.
- **Java:** Follow the [Using JUnit documentation](https://maven.apache.org/plugins-archives/maven-surefire-plugin-2.12.4/examples/junit.html). The assertions for CDKTF specific are available in the `cdktf` package under `Testing`.
- **C#:** Follow the [Getting Started with xUnit.net guide](https://xunit.net/docs/getting-started/netcore/cmdline). The assertions for CDKTF are available in the `cdktf` package under `Testing`.
- **Go:** Follow the [Add a Test guide](https://go.dev/doc/tutorial/add-a-test) in the Go documentation. The assertions for CDKTF specific are available in the `cdktf` package under `Testing`.

After you add testing to your application, run `cdktf init`. CDKTF autogenerates all of the necessary testing files.
```

Reply:
So it is true that all language will have all the generated files needed to start testing, great idea to add a separate section for adding testing to existing project 😁